docs/en/ufs/GCS.md

```diff
@@ -129,7 +129,7 @@ When building your application to use Alluxio, your application should include a
 `alluxio-core-client-fs` module to use the
 [Alluxio file system interface]({{ '/en/api/FS-API.html' | relativize_url }}) or the
 `alluxio-core-client-hdfs` module to use the
-[Hadoop file system interface](https://wiki.apache.org/hadoop/HCFS). For example, if you
+[Hadoop file system interface](https://cwiki.apache.org/confluence/display/HADOOP2/HCFS). For example, if you
```

Comment:
This docs looks really old, I suggest directly remove the whole `Configuring application dependency` section. They are not useful